auto_post_classifier/main.py

```diff
@@ -1,62 +1,64 @@
-import json
+import logging
```

Comment:
Do you need both `logging` and `loguru`?

Reply:
fixed, remove also the 422 response code handle